### PR TITLE
Update binder resources

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by dougbu** [comment]
This PR is incomplete without an update in [eng/PatchConfig.props](https://github.com/aspnet/AspNetCore/blob/release/2.2/eng/PatchConfig.props#L46-L49).

**Comment by vivmishra** [comment]
Approved for 2.1.10 and 2.2.4.
Checkin ASAP.

**Comment by HaoK** [comment]
@dougbu should I open a new PR for 2.2 or should I merge this one from 2.1 to 2.2 after its in?

**Comment by dougbu** [comment]
> open a new PR for 2.2 or should I merge this one from 2.1 to 2.2 after its in?

Wait for the automated 2.1 -> 2.2 PR and then use that.

File: eng/PatchConfig.props [hunk 1]
```diff
@@ -12,4 +12,10 @@
     </PackagesInPatch>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.10' ">
```
Comment (dougbu) [inline eng/PatchConfig.props]: This will conflict w/ my changes to this file for the 2.1.10 branding but I'll deal with it.

**Comment by dougbu** [comment]
Changes here look a little strange but should be :ok: (#1256 is now in). Good to go if you're done @HaoK

**Comment by ajcvickers** [comment]
@dougbu Do you need me to merge, or are we waiting for Hao, or are you going to do it?

**Comment by dougbu** [comment]
@ajcvickers I was waiting for confirmation from @HaoK that he was done. If you know he is, lemme know or Squash&Merge (as you wish).

**Comment by HaoK** [comment]
This is done, feel free to click the button :)

**Pull request merged**